### PR TITLE
refactor: display default token while waiting for network token

### PR DESCRIPTION
**Pull request by dated**
## Proposed changes

Similar to https://github.com/ArkEcosystem/explorer/pull/501, this adds a fallback value for the network token while it is being fetched.

## Types of changes

- [x] Refactoring (improve a current implementation without adding a new feature or fixing a bug)

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -125,8 +125,8 @@
     "webpack-merge": "4.1.4"
   },
   "engines": {
-    "node": ">= 6.0.0",
-    "npm": ">= 3.0.0"
+    "node": ">= 9.0.0",
```
Comment (faustbrian) [inline package.json]: We should raise this to 10 and drop the 9 tests completely on CircleCI and instead run 10/11.

**Review by faustbrian — APPROVED** [review]

File: resources/views/app/block.blade.php [hunk 2]
```diff
@@ -31,7 +31,7 @@
                 <div class="py-16 content-container md:px-8">
                     <div id="transaction-list" class="w-full">
                         <div class="relative flex items-end justify-between mb-8">
-                            <h2 class="text-xl sm:text-2xl">@lang('pages.block.transactions')</h2>
+                            <h2>@lang('pages.block.transactions')</h2>
```
Comment (ItsANameToo) [inline resources/views/app/block.blade.php]: you need to pay attention to the classes that were set. `class="text-xl sm:text-2xl"` doesn't translate to `h2` but to `h4` instead, so you should be using `h4` here

File: resources/views/dashboard.blade.php [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <x-app-layout>
```
Comment (ItsANameToo) [inline resources/views/dashboard.blade.php]: this file can be removed

**Comment by ItsANameToo** [comment]
@samharperpittam can you go over the pages that were changed here to ensure the headings match designs?

**Pull request merged**